custom/icds_reports/migrations/0092_pse_days_21.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2019-01-18 22:39
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+from django.db import migrations
+from corehq.sql_db.operations import RawSQLMigration
+
+migrator = RawSQLMigration(('custom', 'icds_reports', 'migrations', 'sql_templates'))
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0091_lunch'),
```

Comment:
unfortunately I think this will also conflict with migration in #23109

Its a bit difficult to solve this well in django. My general strategy previously has been to structure my PRs such that all the migrations are in one PR, or the PRs build off of one another when there is a some order that they will be merged in. I know that most of the PRs you've submitted can be merged in parallel, so I'm not sure on the best way to structure them so that you don't run into this issue

Reply:
yeah, So I am following a simple not so smart strategy  to do this. Before merging any PR with migration I first merge it master then again run `migrate` in local and see if any conflict is there. if there is a conflict, resolve that and then push 1 commit after resolving that. By this I  ensure that its not conflicting. 
So for this. This is going to conflict if https://github.com/dimagi/commcare-hq/pull/23109 is merged first. Whatever branch will be merged later, I will be performing the above steps to remove the conflict. @emord

Reply:
I think, I Like your Idea of making all the migrations in one PR then do the work later in later PRS, which I can use for  working in any realease work or any coherent work